plugins/outputs/cloudwatch/util.go

```diff
@@ -46,11 +47,25 @@ const (
 	unitOverheads = 42
 )
 
-func publishJitter(publishInterval time.Duration) (publishJitter time.Duration) {
-	r := rand.New(rand.NewSource(time.Now().Unix()))
-	jitter := r.Int63n(int64(publishInterval.Seconds()))
-	publishJitter = time.Duration(jitter) * time.Second
-	return
+// Global set once.
+var doOnce sync.Once
+var gRand *rand.Rand
+
+// publisJitterInt returns a random int64 between 0 and the given maxVal.
+func publisJitterInt(maxVal int64) int64 {
```

Comment:
Typo, but maybe a better name would just be to call it something else entirely. At first glance, I thought we were doing another publish function that jitters with an int. But what this really does is it spits out random values to use as the jitter. I'm bad at names but maybe:
```suggestion
func getJitter(maxVal int64) int64 {
```

Reply:
will fix